pkg/ipam/subnet.go

```diff
@@ -64,6 +68,8 @@ func NewSubnet(name, cidrStr string, excludeIps []string) (*Subnet, error) {
 			V4FreeIPList:     IPRangeList{&IPRange{Start: IP(firstIP), End: IP(lastIP)}},
 			V4ReleasedIPList: IPRangeList{},
 			V4ReservedIPList: convertExcludeIps(v4ExcludeIps),
+			V4AvailIPList:    IPRangeList{&IPRange{Start: IP(firstIP), End: IP(lastIP)}},
+			V4UsingIPList:    IPRangeList{},
 			V4NicToIP:        map[string]IP{},
```

Comment:
the V6AvailIPList and V6UsingIPList can also add here as default value

Reply:
it seems no need to because V6FreeIPList,V6ReleasedIPList... are not set default value in this place either